tests/foreman/api/test_capsulecontent.py

```diff
@@ -1037,16 +1026,20 @@ def test_positive_sync_collection_repo(
             '[galaxy_server.capsule_galaxy]\n'
             f'url={repo_path}\n'
         )
-        rhel7_contenthost.execute(f'echo "{cfg}" > ./ansible.cfg')
+
+        @request.addfinalizer
+        def _cleanup():
+            target_sat.execute(f'rm -rf {cfg_path} {coll_path}')
```

Comment:
```suggestion
        request.addfinalizer(target_sat.execute(f'rm -rf {cfg_path} {coll_path}'))
```

Reply:
Actually I tried it this way at first, but it failed locally so went the other way. 
```
>       request.addfinalizer(target_sat.execute(f'rm -rf {cfg_path} {coll_path}'))
...
status: 0, node = <Function test_positive_sync_collection_repo>

    def addfinalizer(self, finalizer: Callable[[], object], node: Node) -> None:
        """Attach a finalizer to the given node.
    
        The node must be currently active in the stack.
        """
        assert node and not isinstance(node, tuple)
>       assert callable(finalizer)
E       AssertionError
```

Reply:
```suggestion
        request.addfinalizer(lambda: target_sat.execute(f'rm -rf {cfg_path} {coll_path}'))
```

Reply:
Nice, this looks to work locally. :+1: 
Updated and re-triggered the PRT.